src/main/java/duke/commands/ListAll.java

```diff
@@ -0,0 +1,13 @@
+package duke.commands;
+
+
```

Comment:
There is another empty line that can be removed here

Reply:
Yup... Thanks, it's a terrible coding habit. Thanks for pointing out